### ISSUE TITLE
Printing in a Current Linux Environment.

### ISSUE THREAD

**Issue by thomascrisan** [typora/typora-issues#6078]
With the current version of  of  Arch Linux, printing in Typora 1.8.10, printing has become impossible.  The app starts to print and then crashes and disappears from the screen.  This is independent of printer selected.

This may be due to various libraries and dependencies that Typora needs have been updated in Arch Linux and the application may need to be recompiled in a current Linux environment.

Here is result when I start Typora with `typora --dev` and attempt to print:
```
typora --dev
[131987:0621/112401.871881:ERROR:object_proxy.cc(590)] Failed to call method: org.freedesktop.portal.Settings.Read: object_path= /org/freedesktop/portal/desktop: org.freedesktop.DBus.Error.UnknownMethod: No such interface “org.freedesktop.portal.Settings” on object at path /org/freedesktop/portal/desktop

** (Typora:131987): WARNING **: 11:24:11.171: GDBus.Error:org.freedesktop.DBus.Error.NameHasNoOwner: Could not activate remote peer 'org.freedesktop.Avahi': activation request failed: unknown unit

** (Typora:131987): WARNING **: 11:24:11.171: GDBus.Error:org.freedesktop.DBus.Error.NameHasNoOwner: Could not activate remote peer 'org.freedesktop.Avahi': activation request failed: unknown unit
Segmentation fault (core dumped)
```

**Comment by thomascrisan** [comment]
As an update, version 1.9.3 also does not print in an up to date Arch Linux environment.  When printing, I receive a `segmentation fault (core dumped)` error message.

**Comment by abnerlee** [comment]
merge with #5033

What is the Arch Linux version?
Does print function in the binary version https://typora.io/linux/Typora-linux-x64.tar.gz works?

**Comment by thomascrisan** [comment]
I don't understand why this issue was marked as "completed".  I responded to your questions "Does print function in the binary version https://typora.io/linux/Typora-linux-x64.tar.gz works" that it does not and that Arch Linux is a rolling release.  

I did not know the same questions were posted here.  What is the relationship to this Github issue board to Typora Support?

This issue arose about one month ago and I do not  believe it is related to https://github.com/typora/typora-issues/issues/5033.

It's quite disappointing to pay for a commercial product that does not work and to receive essentially no support.
